pages/view/[linkId].tsx

```diff
@@ -20,12 +20,17 @@ export default function ViewPage() {
     );
   }
 
-  const { expiresAt, emailProtected, password: linkPassword } = link;
+  const {
+    expiresAt,
+    emailProtected,
+    password: linkPassword,
+    isArchived,
```

Comment:
The `isArchived` is derived from the `useLink()` custom hook, which in turn invokes an API. Plesae note that the API currently doesn't provide the `isArchived` column. As a result, the `isArchived ` will always  be undefined. To ensure that this code aligns with our expectations, I kindly suggest making the necessary adjustments and testing them locally before proceeding with the pull request. Take you time :)

Reply:
Thanks a lot for the feedback. I'm on to it.

Reply:
I have made all the required changes. Can you please give it a look and update me if any further changes are required?

Reply:
`// If the link is expired, show a 404 page` modify this comment which basically says `// If the link is expired or archived, show a 404 page` and you can resolve the comment. Review is done from my end. @mfts  will review it too and merge if it make sense. Thanks for your contribution :)